scipy-stubs/interpolate/_rgi.pyi

```diff
@@ -52,7 +52,7 @@ class RegularGridInterpolator(Generic[_SCT_co]):
     def __call__(
         self,
         /,
-        xi: onp.ToFloatND,
+        xi: onp.ToFloat | onp.ToFloatND,
```

Comment:
Does this say `xi` can be a float or 0D? If yes then no, it should be at least 1D (a single data point in an N-D space is a 1D array of length N)

Reply:
> Does this say `xi` can be a float or 0D?

exactly


> If yes then no, it should be at least 1D (a single data point in an N-D space is a 1D array of length N)

I'll correct it then 👌🏻